CMakeLists.txt

```diff
@@ -48,4 +56,12 @@ include(CMakePackageConfigHelpers)
 configure_package_config_file(CMake/curlcppConfig.cmake.in
   "${PROJECT_BINARY_DIR}/curlcppConfig.cmake"
   INSTALL_DESTINATION "${CMAKE_INSTALL_LIBDIR}/cmake/curlcpp")
-install(FILES ${PROJECT_BINARY_DIR}/curlcppConfig.cmake DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/curlcpp)
\ No newline at end of file
+install(FILES ${PROJECT_BINARY_DIR}/curlcppConfig.cmake DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/curlcpp)
+
+join_paths(libdir_for_pc_file "\${exec_prefix}" "${CMAKE_INSTALL_LIBDIR}")
+join_paths(includedir_for_pc_file "\${prefix}" "${CMAKE_INSTALL_INCLUDEDIR}")
+configure_file("${PROJECT_SOURCE_DIR}/src/libcurlcpp.pc.in"
+  "${PROJECT_BINARY_DIR}/src/libcurlcpp.pc"
+  @ONLY)
+install(FILES "${PROJECT_BINARY_DIR}/src/libcurlcpp.pc"

```

Comment:
The pkg-config name is `libcurlcpp`, and is used by e.g. `pkg_check_modules(CURLCPP libcurlcpp)` (cmake) or `dependency('libcurlcpp')` (meson).

The cmake name is `curlcpp` and is used by `find_package(curlcpp)` (cmake) or `dependency('curlcpp')` (meson).

For consistency, these should be the same name: curlcpp. That way Meson can do `dependency('curlcpp')` for both.